plugins/LayerView/LayerView.py

```diff
@@ -136,6 +154,71 @@ def setLayer(self, value):
 
             self.currentLayerNumChanged.emit()
 
+    def setMinimumLayer(self, value):
+        if self._minimum_layer_num != value:
+            self._minimum_layer_num = value
+            if self._minimum_layer_num < 0:
+                self._minimum_layer_num = 0
+
+            self._startUpdateTopLayers()
+
+            self.currentLayerNumChanged.emit()
+
+    def setLayerViewType(self, layer_view_type):
+        self._layer_view_type = layer_view_type
+        self.currentLayerNumChanged.emit()
+
+    def getLayerViewType(self):
+        return self._layer_view_type
+
+    def setExtruderOpacity(self, extruder_nr, opacity):
```

Comment:
Documentation + bit more defensive coding. We don't handle the case of trying to set the opacity of extruder_nr "OMGWTFBBQ" (which never exists)